Khan/perseus
# Transition mobile-keypad to use AphroditeCSSTransitionGroup (vendored from webapp)

## Summary:

After setting up analytics for Keypad open/closed events and integrating it into webapp, we noticed that these events were over-firing by quite a bit. Turns out, the mobile keypad renders the full keypad and then animates displaying and hiding it (and toggles the `visibility` style) but never unmounts it. 

This PR changes things so that we import the `AphroditeCSSTransitionGroup` which manages animating in/our components as they are mounted/unmounted. This allows us to only render the Keypad when it's actually active. 

Issue: LC-1346

## Test plan:

- Integrate into webapp.
- Open a browser and enable dev tools and mobile emulation.
- Switch to the **Network** tab in dev tools.
- Navigate to an exercise that contains an `expression` widget.
- Note that there were NO CEDAR events fired for `MathInputVirtualKeypadOpened` or `MathInputVirtualKeypadClosed` during the rendering of the exercise.
- Tap into the expression and note that the `MathInputVirtualKeypadOpened` event fired.
- Close the keypad and note that the `MathInputVirtualKeypadClosed` event fired.
